src/lib/openapi/spec/client-metrics-schema.ts

```diff
@@ -5,18 +5,48 @@ export const clientMetricsSchema = {
     $id: '#/components/schemas/clientMetricsSchema',
     type: 'object',
     required: ['appName', 'bucket'],
+    description:
+        'Client usage metrics, accumulated in buckets of hour by hour by default',
     properties: {
-        appName: { type: 'string' },
-        instanceId: { type: 'string' },
-        environment: { type: 'string' },
+        appName: {
+            description:
+                'The name of the application that is evaluating toggles',
+            type: 'string',
+            example: 'insurance-selector',
+        },
+        instanceId: {
+            description:
+                'A [(somewhat) unique identifier](https://docs.getunleash.io/reference/sdks/node#advanced-usage) for the application',
+            type: 'string',
+            example: 'application-name-dacb1234',
+        },
+        environment: {
+            description: 'Which environment the application is running in',
+            type: 'string',
+            example: 'development',
+        },
         bucket: {
             type: 'object',
             required: ['start', 'stop', 'toggles'],
+            description:
+                'Holds all metrics gathered over a window of time. Typically 1 hour wide',
             properties: {
-                start: { $ref: '#/components/schemas/dateSchema' },
-                stop: { $ref: '#/components/schemas/dateSchema' },
+                start: {
+                    $ref: '#/components/schemas/dateSchema',
+                    description:
+                        'The start of the time window these metrics are valid for. The window is usually 1 hour wide',
+                    example: '1926-05-08T12:00:00.000Z',
+                },
+                stop: {
+                    $ref: '#/components/schemas/dateSchema',
+                    description:
+                        'The end of the time window these metrics are valid for. The window is 1 hour wide',
+                    example: '1926-05-08T13:00:00.000Z',
+                },
                 toggles: {
                     type: 'object',
+                    description:
+                        'an object containing feature names with yes/no plus variant usage',
```

Comment:
This sentence contains a lot of words. I'd suggest either breaking it into several sentences or adding more qualifiers:

```suggestion
                        'An object whose keys are feature names and whose values are contain statistics about their evaluation.',
```